framework/src/Volo.Abp.Http.Client/Volo/Abp/Http/Client/DynamicProxying/ApiDescriptionFinder.cs

```diff
@@ -94,14 +94,32 @@ protected virtual async Task<ApplicationApiDescriptionModel> GetApiDescriptionFr
 
                 var result = JsonConvert.DeserializeObject(
                     content,
-                    typeof(ApplicationApiDescriptionModel),
-                    new JsonSerializerSettings
-                    {
-                        ContractResolver = new CamelCasePropertyNamesContractResolver()
-                    });
+                    typeof(ApplicationApiDescriptionModel), GetSharedJsonSerializerSettings());
 
                 return (ApplicationApiDescriptionModel)result;
             }
         }
+
+        private static readonly object SyncObj = new object();
```

Comment:
No need to make this lazy. Just set it in a static constructor. Like done for JsonLocalizationDictionaryBuilder.cs

Reply:
It has been modified.